docs/migration/5_0.md

```diff
@@ -775,6 +775,10 @@ Due to i18next migration, certain set of keys have been migrated (from `_plural`
 
 - Text type icon symbols are now converted into text, meaning DOM does not render HTML symbols anymore.
 
+### json-ld
```

Comment:
Just a reminder - we should add the schematics migrations for the breaking changes done in the previous PR, as well as the ones done in this PR.

Reply:
schematics for previous PR will be handled in [CXSPA-1491](https://jira.tools.sap/browse/CXSPA-1491)
Add schematics for icon-loader service change